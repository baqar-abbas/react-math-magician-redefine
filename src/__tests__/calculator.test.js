import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Calculator from '../components/Calculator';

test('renders the Calculator component', () => {
  const { container } = render(<Calculator />);
  expect(container).toMatchSnapshot();
});
