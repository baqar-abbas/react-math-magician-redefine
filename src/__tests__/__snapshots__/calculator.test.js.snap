// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Calculator component 1`] = `
<div>
  <h2
    class="calculator-heading"
    style="color: whitesmoke; font-size: 2rem; font-weight: 500; text-align: center; margin-bottom: 6rem; text-shadow: 2px 2px 4px #0000FF;"
  >
    Lets Do Some Math!
  </h2>
  <div
    class="calcDiv"
  >
    <textarea
      class="calcScreen"
      id="calcScreen"
      name="calcScreen"
    >
      0
    </textarea>
    <div
      class="flex-container"
    >
      <button
        type="button"
      >
        AC
      </button>
      <button
        type="button"
      >
        +/-
      </button>
      <button
        type="button"
      >
        %
      </button>
      <button
        class="operators"
        type="button"
      >
        รท
      </button>
      <button
        type="button"
      >
        7
      </button>
      <button
        type="button"
      >
        8
      </button>
      <button
        type="button"
      >
        9
      </button>
      <button
        class="operators"
        type="button"
      >
        x
      </button>
      <button
        type="button"
      >
        4
      </button>
      <button
        type="button"
      >
        5
      </button>
      <button
        type="button"
      >
        6
      </button>
      <button
        class="operators"
        type="button"
      >
        -
      </button>
      <button
        type="button"
      >
        1
      </button>
      <button
        type="button"
      >
        2
      </button>
      <button
        type="button"
      >
        3
      </button>
      <button
        class="operators"
        type="button"
      >
        +
      </button>
      <button
        class="zero"
        type="button"
      >
        0
      </button>
      <button
        type="button"
      >
        .
      </button>
      <button
        class="operators"
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
